src/config.tsx

```diff
@@ -1,3 +1,4 @@
 export const applicationName = 'Utils';
 export const companyName = 'MultiversX';
 export const companyWebsite = 'https://multiversx.com/';
+export const explainerApiUrl = 'https://devnet-tools.multiversx.com/xbot/api/xai';
```

Comment:
This should be computed related to the selected network (`mainnet`, `devnet`, `testnet`).
eg. 
for `mainnet` => `https://tools.multiversx.com/xbot/api/xai;`
for `devnet` => `https://devnet-tools.multiversx.com/xbot/api/xai;`
for `testnet` => `https://testnet-tools.multiversx.com/xbot/api/xai;`

There is a hook named useChain() which returns the selected network. You can use this hook in order to compute the explainerApiURL. An idea would be to store the endpoint as a constant in the config.ts file (`export const explainerApiUrl = "/xbot/api/xai"`) and compute only the BASE URL